image/verify/fuzz/Cargo.toml

```diff
@@ -0,0 +1,54 @@
+# Licensed under the Apache-2.0 license
+
+[package]
+name = "caliptra-image-verify-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2021"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+libfuzzer-sys = { version = "0.4.6", optional = true }
+afl = { version = "0.13.3", optional = true }
+
+[patch.crates-io]
+byteorder = { git = "https://github.com/benjamindoron/byteorder.git", branch = "struct_aware-1.4.3" }
+zerocopy = { git = "https://github.com/benjamindoron/zerocopy.git", branch = "struct_aware-v0.6.3" }
+
+[dependencies.caliptra-lms-types]
+path = "../../../lms-types"
+features = ["arbitrary"]
+
+[dependencies.caliptra-image-types]
+path = "../../types"
+features = ["arbitrary"]
```

Comment:
AFAICT, caliptra-image-types doesn't have an arbitrary feature:

```
[features]
default = ["std"]
std = []
```

Reply:
Hmm, apparently cargo automatically adds features for optional deps. That's news to me:

https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies

Reply:
Optional dependencies have implicitly defined features to enable them.